README.md

```diff
@@ -5,6 +5,9 @@
 The Hedera Transaction Tool application is a demo application that allows a user to generate keys, create, sign, and submit transactions to a Hedera network. This software is designed for use solely by the Hedera Council and staff. The software is being released as open source as example code only, and is not intended or suitable for use in its current form by anyone other than members of the Hedera Council and Hedera personnel. If you are not a Hedera Council member or staff member, use of this application or of the code in its current form is not recommended
 and is at your own risk.
 
+## Documentation
+The user guide for this application can be found [here][https://docs.hedera.com/hedera-transactions-tool).
```

Comment:
Add a newline between heading and text.

```suggestion

The user guide for this application can be found [here][https://docs.hedera.com/hedera-transactions-tool).
```

Reply:
Newline is not required, so not sure why that's suggested here.
There _is_ an incorrect character, however ( `(` instead of `[` ):
```suggestion

The user guide for this application can be found [here](https://docs.hedera.com/hedera-transactions-tool).
```

Reply:
The rest of the file followed the structure Heading / newline / content, so I was proposing following the existing format.

Reply:
Fair point.  I was just curious on that.

Reply:
Thanks for the quick reviews on this! Will get that updated.

Reply:
@SimiHunjan there's another issue that @jsync-swirlds found - `(` instead of `[` by the URL.

Reply:
omg lol I don't know how I missed that since it didn't render correctly. Thanks all updated!